Example/cocoapods/Podfile

```diff
@@ -1,15 +0,0 @@
-source 'https://cdn.cocoapods.org/'
```

Comment:
@OhKanghoon , removing the cocoapod support is not an option. Pod integration is still used by some developers. It must continue to work.

Reply:
I don't intend to remove the CocoaPods validation code either.
We are utilizing the Podfile in the root directory, and actual building and testing are done using xcworkspace.
This code removal is to avoid managing duplicate Podfiles.